project_name/settings.py-tpl

```diff
@@ -66,6 +67,8 @@ MIDDLEWARE = [
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "django_structlog.middlewares.RequestMiddleware",
+    "csp.middleware.CSPMiddleware",
+    "{{ project_name }}.middleware.XSSFilteringMiddleware",
```

Comment:
Should this change also include a middleware.py-tpl file?

Reply:
Yep, definitely didn't check that file in 🤦‍♂️